docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+
+```bash
+gnokey query vm/qfuncs --data "gno.land/r/demo/wugnot" -remote https://rpc.gno.land:443
+```
+
+The output is a string containing all exported functions for the `wugnot` realm:
+
+```json
+height: 0
+data: [
+        {
+          "FuncName": "Deposit",
+          "Params": null,
+          "Results": null
+        },
+        {
+          "FuncName": "Withdraw",
+          "Params": [
+            {
+            "Name": "amount",
+            "Type": "uint64",
+            "Value": ""
+            }
+          ],
+          "Results": null
+        },
+        // other functions
+]
+```
+
+### `vm/qfile`
+
+With the `vm/qfile` query, we can fetch files found on a specific package path.
+To specify the path we want to query, we can use the `-data` flag:
+
+```bash
+gnokey query vm/qfile -data "gno.land/r/demo/wugnot" -remote https://rpc.gno.land:443
+```
+
+The output is a string containing all exported functions for the
+`wugnot` realm:
+
+```bash
+height: 0
+data: gno.mod
+wugnot.gno
+z0_filetest.gno
+```
+
+### `vm/qeval`
+
+`vm/qeval` allows us to evaluate a call to an exported function without using gas,
+in read-only mode. For example:
+
+```bash
+gnokey query vm/qeval -remote https://rpc.gno.land:443 -data "gno.land/r/demo/wugnot
+BalanceOf(\"g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5\")" 
+```
+
+This command will return the `wugnot` balance of the above address without using gas.
+Properly escaping quotation marks, and inputting a new line for the function
+is currently required.
+
+### `vm/qrender`
+
+`vm/qrender` is an alias for executing `vm/qeval` on the `Render("")` function.
+We can use it like this:
+
+```bash
+gnokey query vm/qrender --data "gno.land/r/demo/userbook
+" -remote https://rpc.gno.land:443
+```
+
+Running this command will display the current `Render()` output of the Userbook
+realm, which is also displayed by default on the [realm's page](https://gno.land/r/demo/userbook):
+
+```bash
+height: 0
+data: # Welcome to UserBook!
+
+## UserBook - Page #1:
+
+#### User #0 - g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 - signed up at Block #0
+#### User #1 - g125em6arxsnj49vx35f0n0z34putv5ty3376fg5 - signed up at Block #0
+#### User #2 - g1urt7pdmwg2m6z3rsgu4e8peppm4027fvpwkmj8 - signed up at Block #0
+#### User #3 - g1uf8u5jf2m9l80g0zsfq7tufl3qufqc4393jtkl - signed up at Block #0
+#### User #4 - g1lafcru2z2qelxr33gm4znqshmpur6l9sl3g2aw - signed up at Block #0
+---
+
+#### Total users: 5
+#### Latest signup: User #4 at Block #0
+---
+
+You're viewing page #1
+```
+
+## Making an airgapped transaction
+
+`gnokey` provides a way to create a transaction, sign it, and later
+broadcast it to a chain in an airgapped manner. With this approach, while it is
+more complicated, users can get full control over the creation, signing and
+broadcasting process of transactions.
+
+Here are the steps taken in this process:
+1. Fetching account information from the chain
```

Comment:
I think the imperative form reads better here.

The steps are:
1. Fetch account...
2. Create an...
3. Sign the..
4. Broadcast the...

Reply:
[844f301](https://github.com/gnolang/gno/pull/2226/commits/844f301d47eca9a077d43bf5ddc48c07177678a0)